collections/_topics/klimaticka-zmena.md

```diff
@@ -11,6 +11,168 @@ intro: |
 
   **Dopady zmeny klímy** na spoločnosť aj prírodu, s ktorými sa budeme stretávať v nasledujúcich desaťročiach, **budú priamo závislé od množstva skleníkových plynov, ktoré ešte do atmosféry vypustíme**, či už spaľovaním fosílnych palív alebo inými aktivitami, pri ktorých vzniká veľké množstvo emisií.
 
+dashboard:
+- type:        "single"
+  col-lg-size: "3-5"
+  col-xl-size: 3
+  value:       "**420** ppm"
+  subtitle:    "**koncentrácia CO<sub>2</sub>** v atmosfére v roku 2022"
+  # comment:     "(celých 10 000 rokov pred priemyselnou revolúciou bola táto koncentrácia stabilne ~270 ppm)"
+  source:      "Scripps Institution of Oceanography, University of California"
+  source-url:  "https://keelingcurve.ucsd.edu/"
+- type:        "single"
+  col-size:    6
+  col-lg-size: "3-5"
+  col-xl-size: 3
+  value:       "**+ 1,2 °C**"
+  subtitle:    "**oteplenie sveta** od druhej polovice 19. storočia"
+  # comment:     "(to je asi +0,2 °C za dekádu)"
+  source:      "NASA, Goddard Institute for Space Studies"
+  source-url:  "/teplotna-anomalia"
+- type:        "single"
+  col-size:    6
+  col-lg-size: "3-5"
+  col-xl-size: 3
+  value:       "**+ 1,8 °C**"
+  subtitle:    "**oteplenie Slovenska** od roku 1881"
+  # comment:     "(to je asi asi +0,13 °C za dekádu)"
+  source:      "Ministerstvo životného prostredia SR"
+  source-url:  "https://www.minzp.sk/files/odbor-politiky-zmeny-klimy/strategia-adaptacie-sr-zmenu-klimy-aktualizacia.pdf"
+- type:        "compare"
+  col-lg-size: 5
+  col-xl-size: "4-5"
+  title:       "**Zaľadnenie Severného ľadového oceánu**"
+  data:
+  - value:     "**7,5** mil. km²"
+    subtitle:  "v septembri 1980"
+  - value:     "**4,7** mil. km²"
+    subtitle:  "v septembri 2021"
+  source:      "NASA, satelitné merania"
+  source-url:  "https://climate.nasa.gov/vital-signs/arctic-sea-ice/"
+- type:        "compare"
+  col-lg-size: 5
+  col-xl-size: "4-5"
+  title:       "**Zvýšenie hladín oceánov** od roku 1900"
+  data:
+  - value:     "**20** cm"
+    subtitle:  "do roku 2018"
+  - value:     "**80–150** cm"
+    subtitle:  "očakávané do roku 2150"
+  source:      "Súhrnná správa IPCC WG1, Technical Summary"
+  source-url:  "https://www.ipcc.ch/report/ar6/wg1/downloads/report/IPCC_AR6_WGI_TS.pdf#page=45"
+
+subtopics:
+  # Klimatická zmena v kocke
+- lead: |
+    Ako sa vyznať v zložitej problematike klímy a jej zmien? Pre začiatok je určite dobré vedieť, že:
+     commented-content:
```

Comment:
tohle máš špatně odsazené a shazuje vám to build, commented-content musí být na stejné úrovni odsazení jako "lead:"
takhle je to na vaší české verzi:
https://github.com/faktaoklimatu/web-cz/blob/master/collections/_topics/klimaticka-zmena.md?plain=1#L68